src/Tools/dotnet-dump/Dumper.cs

```diff
@@ -86,6 +86,12 @@ public int Collect(IConsole console, int processId, string output, bool diag, Du
 
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 {
+                    if (crashreport)
```

Comment:
How is this not supported on Windows? I thought crash reports were possible on Windows. I seen many programs use such a thing before there too.

Reply:
The crash report referenced here is a special json file that the Linux and MacOS tool createdump tool generates.  The Windows "createdump" just calls the OS's MiniDumpWriteDump to generate dump and doesn't have a way to generate this json file. On Linux and MacOS createdump explicitly gathers the runtime process's state and writes both the core dump and the crash report json from that data.